go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	google.golang.org/protobuf v1.25.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/ini.v1 v1.57.0 // indirect
+	gopkg.in/src-d/go-log.v1 v1.0.1
```

Comment:
why does this appear here again?

Reply:
please rebase and go mod tidy if necessary. this branch still contains the bad logger stuff from the deploy pr.

Reply:
I reverted this change but I actually don't see this as a problem. Dunno where its coming from but the build is fine